InfiniteScroll/InfiniteScroll/ViewController.swift

```diff
@@ -8,12 +8,58 @@
 import UIKit
 
 class ViewController: UIViewController {
-
+    
+    @IBOutlet weak var myTableView: UITableView!
+    let arr = Array<Int>(0...200)
+    var actualData: [Int] = []
+    let limit: Int = 123
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        
+        myTableView.delegate = self
+        myTableView.dataSource = self
+        myTableView.prefetchDataSource = self
+        
+        actualData = Array<Int>(0...20)
     }
+    
+    func update() {
+        // 함수가 호출될 때 마다 actualData에 10개씩 추가되는 로직
+        guard let last = actualData.last else { return }
+        
+        if (limit + 1) != actualData.count {
```

Comment:
```suggestion
        if limit != actualData.count - 1 {
```

index값의 차이를 매꾸기 위해 +-1을 한다는 의미로 암묵적으로 `count - 1` 을 해주는 것이 자연스럽습니다!

Reply:
`StoryViewController`의 `update()` 함수와 같이 작성해주시면 됩니다.
비슷한 로직이 반복적으로 사용될 때는 한번 더 체크해서 작성하면 좋을 것 같습니다ㅎㅎ